### PR TITLE
Add proposed RHEA mappings which have no logical conflicts.

### PR DESCRIPTION
**Pull request by balhoff**
@hdrabkin here are 1004 RHEA additions which don't have any logical conflicts found by boomer.

### CODE REVIEW

**Comment by hdrabkin** [comment]
But the first one I look at 
@@ -44150,6 +44265,7 @@ synonym: "smooth-muscle-myosin-light-chain kinase activity" NARROW [EC:2.7.11.18
 synonym: "STK18" RELATED [EC:2.7.11.18]
 xref: EC:2.7.11.18
 xref: MetaCyc:2.7.11.18-RXN
+xref: RHEA:22004
 is_a: GO:0004674 ! protein serine/threonine kinase activity
the parent (4674) has two rheas ; I'll need to look at that.

However, this one is troubling because the GO:0045290 def  AND the metacyc uses both NAD or NADP, and uses EC:1.1.1.117, which also uses NAD(P).
but the RHEA only uses NADP ; 
I see that this is a parent term to one term GO:00478:, which is the reaction using just NAD ( and has a different EC:1.1.1.116, with just NAD.
So bottom line is the RHEA suggested for GO:0045290 is more specific than the term. I'm surprised 

Can we make script to see how many defs have NAD(P) in them. In GO I think there are 141 total (OBOEDIT on GO-Basic)

**Comment by balhoff** [comment]
@hdrabkin these GO terms have "NAD(P)" in their definition, and have a RHEA cross reference (either in my PR or currently in the ontology):

```
GO:0000254
GO:0003955
GO:0003956
GO:0004074
GO:0004128
GO:0004489
GO:0008942
GO:0009013
GO:0015045
GO:0015049
GO:0016174
GO:0016509
GO:0018455
GO:0018635
GO:0018671
GO:0033735
GO:0034785
GO:0036190
GO:0036191
GO:0036192
GO:0036193
GO:0045290
GO:0047952
GO:0050492
GO:0050578
GO:0050600
GO:0050601
GO:0051745
GO:0051903
GO:0052601
GO:0052662
GO:0052663
GO:0102564
GO:0102601
GO:0102960
```

**Comment by hdrabkin** [comment]
Great; Ok, how many just in your PR?

Once I know those, if we push it then I can go in and make the extra terms and move the mapping to the children.
hjd
(last night I was going through removing 3 digit ec from Pefein’s enzymes; got about 50 done. Will do more tonight.

**Comment by balhoff** [comment]
I think these are the new ones:

```
GO:0003955 RHEA:46164
GO:0003956 RHEA:19149
GO:0004074 RHEA:15793
GO:0004128 RHEA:46680
GO:0009013 RHEA:13213
GO:0015045 RHEA:13949
GO:0016509 RHEA:52656
GO:0018671 RHEA:19473
GO:0034785 RHEA:35307
GO:0045290 RHEA:21892
GO:0050492 RHEA:21412
GO:0050578 RHEA:35735
GO:0051745 RHEA:24488
GO:0102564 RHEA:48728
GO:0102601 RHEA:55440
```

**Comment by hdrabkin** [comment]
Great thanks; I think we should go ahead;
